google-sheets-append-table.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?><service-task-definition>
 <license>(C) Questetra, Inc. (MIT License)</license>
 <engine-type>1</engine-type>
-<label>New Record to Google Sheets (Table)</label>
-<label locale="ja">Google Sheets 行追加 (Table)</label>
-
-<last-modified>2020-02-06</last-modified>
-<help-page-url>https://support.questetra.com/ja/addons/googlesheets-appendtable/</help-page-url>
-<help-page-url locale="ja">https://support.questetra.com/addons/googlesheets-appendtable/</help-page-url>
+<label>Google Sheets: Append New Rows (Table type data)</label>
+<label locale="ja">Google スプレッドシート: 行追加 (テーブル型データ)</label>
+<summary>Add contents of table data item after the last row in a sheet, inserting new rows into the sheet if necessary.</summary>
```

Comment:
Appends values of table data item after the last row in a sheet. Inserts new rows if necessary.